packages/components/pie-modal/README.md

```diff
@@ -39,3 +39,14 @@ import { PieModal } from '@justeattakeaway/pie-button';
 // react
 import { PieModal } from '@justeattakeaway/pie-button/dist/react';
 ```
+
+## Testing
+
+### Visual tests
+
+Run at the root of the monorepo:
+```
+yarn test:visual --filter=pie-modal
+```
+
+Note: To run these locally, you will need to ensure that any environment variables required are set up on your machine to mirror those on CI (such as Percy tokens). How you achieve this will differ between operating systems.
```

Comment:
I think it would be beneficial to explicitly state how we do this for each machine.